README.md

```diff
@@ -6,22 +6,26 @@ Monorepo for **open-source** Kong UI components and utilities.
 
 [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
 
-- [What goes here](#what-goes-here)
-- [Creating a package](#creating-a-package)
-- [Package Publishing](#package-publishing)
-- [Development](#development)
-  - [Requirements](#requirements)
-  - [Setup](#setup)
-  - [Dev Server](#dev-server)
-  - [Stylelint](#stylelint)
-  - [ESLint](#eslint)
-  - [Type Checking](#type-checking)
-  - [Testing](#testing)
-  - [Preview sandbox build](#preview-sandbox-build)
-  - [Build for production](#build-for-production)
-  - [Committing Changes](#committing-changes)
-- [Preview components](#preview-components)
-- [Running consuming application with local copy of the package](#running-consuming-application-with-local-copy-of-the-package)
+- [public-ui-components](#public-ui-components)
```

Comment:
I think this is the markdown linter not respecting the settings. Can you maybe revert these and only add what's needed at the same levels that exist?

Reply:
yup, dropped it.